spec/rails_helper.rb

```diff
@@ -0,0 +1,24 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+
+require 'rspec/rails'
+require 'spec_helper'
+
+Dir[Rails.root.join('spec/support/*.rb')].each { |file| require file }
+
+SimpleCov.start
+
+begin
+  ActiveRecord::Migration.maintain_test_schema!
+rescue ActiveRecord::PendingMigrationError => e
```

Comment:
standard naming for exceptions is `e`. Actually, `rubocop` is probably complaining about this. Run `rake code_analysis`